src/types.ts

```diff
@@ -4,6 +4,13 @@ export type MeasureStartMessage = {
   staves?: number;
 };
 
+export type PartStartMessage = {
+  msgType: 'partStart';
+  id: string;
+  ndx: number;
+  from: number;
```

Comment:
nit: Based on its usage, this `from` property references the `ndx`. Would you indicate that in the property name?

Reply:
Waht name do you recommend? ndx will be number

Reply:
maxNumber, total, ??

Reply:
I went for msgIndex and msgCount.